### PR TITLE
dev added text

### PR DESCRIPTION
**Pull request by david-m-juneau**
This is a test

### CODE REVIEW

File: feature.md [hunk 1]
```diff
@@ -1 +1,2 @@
 test
+the developer added this text
```
Comment (david-m-juneau) [inline feature.md:2]: This file needs to be reviewed.

Comment (david-m-juneau83) [reply feature.md:2]: Looks good to me.

Comment (david-m-juneau) [reply feature.md:2]: Made a small update. Please review again.

Comment (david-m-juneau83) [inline feature.md:2]: What about...

Comment (david-m-juneau83) [reply feature.md:2]: Done!

**Comment by david-m-juneau83** [comment]
Review complete and updates made. Merging into master...

**Pull request merged**